src/pydot/core.py

```diff
@@ -568,7 +568,7 @@ def set_parent_graph(self, parent_graph: Optional["Common"]) -> None:
         self.obj_dict["parent_graph"] = parent_graph
 
     def get_parent_graph(self) -> Optional["Graph"]:
-        return self.obj_dict.get("parent_graph", None)  # type: ignore
+        return self.obj_dict.get("parent_graph", None)
```

Comment:
I'm surprised this is OK without the ignore, because the _setter_ defines its `parent_graph` input as `Optional["Common"]`. So, returning (what's potentially been set to) an `Optional["Common"]` as an `Optional["Graph"]` feels like it shouldn't work. But, if MyPy is OK with it... `¯\_(ツ)_/¯`

Reply:
Yeah, see, mypy complained about it:

```text
src/pydot/core.py:571: error: Returning Any from function declared to return "Optional[Graph]"  [no-any-return]
```